Sources/ATTNParameterValidation.m

```diff
@@ -36,6 +44,18 @@ + (void)verifyStringOrNil:(nullable NSString *)inputValue inputName:(nonnull con
   }
 }
 
++ (void)verifyString:(nullable NSString *)inputValue inputName:(nonnull const NSString *)inputName {
```

Comment:
can we just add this code to `verifyStringOrNil` instead of creating a new method?

Reply:
yup, this is an internal class so you can change the existing code! and you can fix/add tests to verify as well